packages/back-end/src/api/openapi/paths/postForkEnvironment.yaml

```diff
@@ -0,0 +1,54 @@
+summary: Creates a new environment as a fork of an existing environment
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        required:
+          - id
+          - forkBase
+          - synchronous
+        properties:
+          id:
+            type: string
+            description: The ID of the new environment
+          forkBase:
+            type: string
+            description: The ID of the environment to fork from
+          synchronous:
+            type: boolean
+            description: Whether this fork should be run synchronously. If true, the server may take a long time to respond, particularly if there are many features with rules in the forkBase environment.
```

Comment:
I'd like some input on this param. I opted to let the user decide whether they want this to be sync or async because the performance will depend heavily on how many features they have with rules to be forked. I'd like to default it to sync but that could lead to unacceptably slow performance for this endpoint. Ideally when running it async there'd be a way to check the progress but I also felt like adding that would be overcomplicating this feature.